framework/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Yii Framework 2 Change Log
 - Enh #12881: Added `removeValue` method to `yii\helpers\BaseArrayHelper` (nilsburg)
 - Enh #12901: Added `getDefaultHelpHeader` method to the `yii\console\controllers\HelpController` class to be able to override default help header in a class heir (diezztsk)
 - Enh: Added constants for specifying `yii\validators\CompareValidator::$type` (cebe)
+- Enh: Use ArrayHelper::getValue in SluggableBehavior (thyseus)
```

Comment:
Need issue `#` here. If there's no issue existing for it, use pull request number i.e. `Enh #13035`.